DQM/TrackingMonitor/src/dEdxAnalyzer.cc

```diff
@@ -25,7 +25,7 @@ dEdxAnalyzer::dEdxAnalyzer(const edm::ParameterSet& iConfig)
   , conf_    (fullconf_.getParameter<edm::ParameterSet>("dEdxParameters") )
   , doAllPlots_  ( conf_.getParameter<bool>("doAllPlots") )
   , doDeDxPlots_ ( conf_.getParameter<bool>("doDeDxPlots") )    
-  , genTriggerEventFlag_( new GenericTriggerEventFlag(conf_,consumesCollector(), *this) )
+  , genTriggerEventFlag_( new GenericTriggerEventFlag(conf_.getParameter<edm::ParameterSet>("genericTriggerEventPSet"),consumesCollector(), *this) )
```

Comment:
In principle `genTriggerEventFlag_` could be made a regular member too (or, by quick look I don't see any specific reason why it should be a pointer), which would simplify the construction/destruction side even more.